content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,556 @@
+  you must log in with that user's credentials.
+</Callout>
+
+## Example usage
+
+Here are a few examples demonstrating common uses and capabilities of
+the SDKs.
+
+### Logging in
+
+**API**: Marketplace API
+
+Some endpoints require that a user is logged in first, e.g. creating
+listings, viewing their own listings, or initiating inquiry
+transactions.
+
+```jsx
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+var sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+});
+
+sdk
+  .login({ username: '<User email>', password: '<User password>' })
+  .then((_) => {
+    // Request dependent on an authenticated user access token
+  });
+```
+
+### Initiating a transaction
+
+**API**: Marketplace API
+
+The process of initiating a transaction is highly variable because it
+depends entirely on transaction flow customizations and the required
+parameters of the transition.
+
+For this example, we'll use the default-booking process, and choose
+`transition/request-payment` as the first step. This requires that:
+
+- The customer is logged in
+- A token exchange is made to get a trusted user access token
+- The request includes the following parameters
+  - `transition`
+  - `processAlias`
+  - `params`
+    - `listingID`
+    - `bookingStart`
+    - `bookingEnd`
+    - `lineItems`
+
+<Callout type="info">
+  This example uses the **minimum** required parameters per the
+  transition defined in the transaction process. You'll likely want to
+  include more information during this step, e.g. updating the
+  transaction's extended data. See the [Marketplace API reference
+  documentation](https://www.sharetribe.com/api-reference/marketplace.html#initiate-transaction)
+  for more information.
+</Callout>
+
+```jsx
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+const transition = 'transition/request-payment';
+const processAlias = 'default-booking/release-1';
+
+const params = {
+  listingId: '<Listing ID>',
+  bookingStart: '2025-12-20T22:00:00.000Z',
+  bookingEnd: '2025-12-21T22:00:00.000Z',
+  lineItems: [
+    {
+      code: 'line-item/night',
+      unitPrice: {
+        _sdkType: 'Money',
+        amount: 5000,
+        currency: 'EUR',
+      },
+      quantity: 1,
+      includeFor: ['customer', 'provider'],
+    },
+    {
+      code: 'line-item/provider-commission',
+      unitPrice: {
+        _sdkType: 'Money',
+        amount: 5000,
+        currency: 'EUR',
+      },
+      percentage: -13,
+      includeFor: ['provider'],
+    },
+  ],
+};
+
+const sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+  clientSecret: '<Client secret>',
+});
+
+sdk
+  .login({ username: '<User email>', password: '<User password>' })
+  .then((_) => {
+    sdk.exchangeToken().then((res) => {
+      var store = sharetribeSdk.tokenStore.memoryStore();
+      store.setToken(res.data);
+
+      var sdkTrusted = sharetribeSdk.createInstance({
+        clientId: '<Client ID>',
+        tokenStore: store,
+      });
```

Comment:
This is slightly different than the one you used above where the token store creation was inline inside the SDK creation, i.e. `tokenStore: sharetribeSdk.tokenStore.memoryStore(res.data)`. Maybe use the same here so that it's easier to see the connection between the two?